src/_posts/2025-01-30-new-year-new-pycon.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author: The PyConUK team
+title: New Year, New PyCon UK!
+---
+
+{% assign links = site.data.links.md.short %}
+
+We’re very excited to be kicking off 2025 with an exciting announcement… PyCon
+UK is back, and we’re moving to a brand-new location!
+
+After several fantastic years in Cardiff, the 2025 conference is heading up
+north. We’re delighted to announce that PyCon UK 2025 will be held in
+Manchester from the 19th to the 22nd of September. Known for its rich
+industrial history, vibrant culture, and thriving tech scene, Manchester
+provides the perfect backdrop for the conference.
+
+{% if site.venu_confirmed == true %}
```

Comment:
Typo here, should be venue_confirmed.

Also, I'm wondering if it's worth having the venue_confirmed stuff- if we're looking at being able to confirm it in the next few days then it's probably not? I've asked in the channel (as you've probably seen).

Reply:
I was in half a mind about the conditional stuff too. On reflection, I think I will remove it.